.github/workflows/tests_scripts.yml

```diff
@@ -33,6 +37,48 @@ jobs:
       run: |
         ./install_apptainer_ubuntu.sh
 
+    - name: test load_easybuild_module.sh script
+      run: |
+        export TMPDIR=$(mktemp -d)
+
+        # run tests from /tmp where scripts were copied into,
+        # since scripts must be accessible from within build container
+        cp -a * /tmp/
+        cd /tmp
+
+        for EB_VERSION in '4.5.0' '4.5.1' '4.7.2'; do
+          # Create script that uses load_easybuild_module.sh which we can run in compat layer environment
+          # note: Be careful with single vs double quotes below!
+          #       ${TMPDIR} and ${EB_VERSION} should be expanded, so use double quotes;
+          #       For statements using variables that are only defined in the script, like ${EASYBUILD_INSTALLPATH},
+          #       use single quotes to avoid expansion while creating the script.
+          test_script="${TMPDIR}/eb-${EB_VERSION}.sh"
+          echo '#!/bin/bash' > ${test_script}
+          # both $EB and $TMPDIR environment must be set, required by load_easybuild_module.sh script
+          echo 'export EB="eb"' >> ${test_script}
+          echo "export TMPDIR=${TMPDIR}" >> ${test_script}
+          # set up environment to have utility functions in place that load_easybuild_module.sh script relies on,
+          # along with $EESSI_* environment variables, and Lmod
+          echo 'source /tmp/scripts/utils.sh' >> ${test_script}
+          echo 'source /tmp/init/eessi_environment_variables' >> ${test_script}
+          echo 'source ${EPREFIX}/usr/share/Lmod/init/bash' >> ${test_script}
+          # minimal configuration for EasyBuild so we can test installation aspect of load_easybuild_module.sh script
+          echo "export EASYBUILD_INSTALLPATH=${TMPDIR}" >> ${test_script}
+          echo 'module use ${EASYBUILD_INSTALLPATH}/modules/all' >> ${test_script}
+          echo '' >> ${test_script}
+          echo "source /tmp/load_easybuild_module.sh ${EB_VERSION}" >> ${test_script}
+          echo 'module list' >> ${test_script}
+          echo 'eb --version' >> ${test_script}
+
+          chmod u+x ${test_script}
+
+          # run wrapper script + capture & check output
+          out="${TMPDIR}/eb-${EB_VERSION}.out"
+          ./build_container.sh run /tmp/$USER/EESSI /tmp/run_in_compat_layer_env.sh ${test_script} 2>&1 | tee ${out}
```

Comment:
Should use `eessi_container.sh` 😛

Reply:
Yeah, good point, fixed in abb931c (was trickier than I anticipated!)